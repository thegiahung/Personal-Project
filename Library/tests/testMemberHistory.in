member history 100000
add member Python
member history 100000
member history 111111
add book books.csv 111111
rent 100000 111111
relinquish 100000 111111
member history 100000
EXIT
