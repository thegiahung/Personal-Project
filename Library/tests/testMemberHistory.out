user: No members in system.

user: Success.

user: No rental history for member.

user: No such member in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
